### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by WalkQuackBack**
Updates README for better discovery/SEO, and to explain more clearly what the repository is for. I was unsure whether I should link to the Documentation page but can still change that.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,4 +1,43 @@
...
+  <a href="https://github.com/TKMM-Team/Tkmm/releases">
+    <img src="https://img.shields.io/github/v/tag/TKMM-Team/Tkmm?style=for-the-badge&logoColor=C71B42&color=C71B42&labelColor=2A2C33&logo=git&label=Version" alt="Releases'"/>
+  </a> &nbsp;
+  <a href="https://discord.com/invite/3tNUBWQRXv">
+    <img src="https://img.shields.io/discord/1179611100183011429?style=for-the-badge&logoColor=3b83c8&color=3b83c8&labelColor=2A2C33&logo=discord&label=discord" alt="Discord"/>
+  </a> &nbsp;
+  <a href="https://github.com/TKMM-Team/Tkmm">
+    <img src="https://img.shields.io/github/stars/TKMM-Team/Tkmm?style=for-the-badge&logoColor=FFCB41&color=FFCB41&labelColor=2A2C33&logo=github" alt="Stars"/>
+  </a>
+</p>
```
Comment (ArchLeaders) [inline Readme.md:13]:
```suggestion
<a href="https://github.com/TKMM-Team/Tkmm/releases">
  <img src="https://img.shields.io/github/v/tag/TKMM-Team/Tkmm?style=for-the-badge&logoColor=C71B42&color=C71B42&labelColor=2A2C33&logo=git&label=Version" alt="Releases'"/>
</a> &nbsp;
<a href="https://discord.com/invite/3tNUBWQRXv">
  <img src="https://img.shields.io/discord/1179611100183011429?style=for-the-badge&logoColor=3b83c8&color=3b83c8&labelColor=2A2C33&logo=discord&label=discord" alt="Discord"/>
</a> &nbsp;
<a href="https://github.com/TKMM-Team/Tkmm">
  <img src="https://img.shields.io/github/stars/TKMM-Team/Tkmm?style=for-the-badge&logoColor=FFCB41&color=FFCB41&labelColor=2A2C33&logo=github" alt="Stars"/>
</a>
```

Looks a bit odd center aligned.

File: Readme.md [hunk 2]
```diff
@@ -1,4 +1,43 @@
...
 
-[![License](https://img.shields.io/badge/License-MIT-blue.svg)](License.txt) [![Downloads](https://img.shields.io/github/downloads/TKMM-Team/Tkmm/total)](https://github.com/TKMM-Team/Tkmm/releases)
+---
+
+TKMM is a mod manager and merger for Tears of the Kingdom developed by [TKMM-Team](./about#the-team).
```
Comment (ArchLeaders) [inline Readme.md:17]:
```suggestion
TKMM is a mod manager and merger for Tears of the Kingdom developed by [TKMM-Team](https://totkmods.github.io/tkmm/about#the-team).
```

This relative link doesn't exist.

File: Readme.md [hunk 3]
```diff
@@ -1,4 +1,43 @@
...
+
+If you wish to show your support for the project, please leave a star on the [GitHub repository](https://github.com/TKMM-Team/Tkmm).
```
Comment (ArchLeaders) [inline Readme.md:19]:
```suggestion
```

We don't need to link back to this page.

File: Readme.md [hunk 4]
```diff
@@ -1,4 +1,43 @@
...
+
+## Download TKMM
+
+<div id="download-btns">
+  <a class="download-btn" id="download-all" href="./downloads/">Downloads</a>
```
Comment (ArchLeaders) [inline Readme.md:24]:
```suggestion
  <a class="download-btn" id="download-all" href="https://totkmods.github.io/tkmm/downloads/">Downloads</a>
```

This relative link doesn't exist.

**Review by ArchLeaders — CHANGES_REQUESTED** [review]

**Comment by WalkQuackBack** [comment]
Merged requested changes

**Review by ArchLeaders — APPROVED** [review]
Looks good, thanks.

**Pull request merged**